data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/http.mapping.yml

```diff
@@ -0,0 +1,11 @@
+pluginName: http
+mappedAttributes:
+  port: port
+  max_pending_requests: max_pending_requests
+  threads: thread_count
+  ssl: ssl
+  ssl_key: ssl_key_file
+  ssl_certificate: ssl_certificate_file
+additionalAttributes:
```

Comment:
Where do these two constants come from? `500` and `10_000`?

Reply:
https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/http-source#configurations

Reply:
I thought we were just mapping the names here and not the default values?

Reply:
If we need to map a value from Logstash, I think this would be appropriate. But, it sounds like these are default values in Data Prepper's HTTP plugin. I think those types of defaults should not be mapped.

Reply:
I would agree. If it isn't mentioned in the Logstash config, then the default will be used by Data Prepper plugins.

Reply:
These are not mapped, they are part of the `additionalAttributes`. But considering these attributes are optional I can get rid of them too.